combat-trainer.lic

```diff
@@ -333,6 +333,14 @@ class LootProcess
     echo("  @last_rites: #{@last_rites}") if $debug_mode_ct
     @last_rites_timer = Time.now - 600
 
+    if settings.box_loot_limit
+      @box_nouns = get_data('items').box_nouns
+      @box_loot_limit = settings.box_loot_limit
+      @current_box_count = get_boxes(settings.picking_box_source).size + get_boxes(settings.picking_pet_box_source).size
```

Comment:
If these two sources are the same container, will this double your number of boxes?

Reply:
hmm good catch.